types/devexpress-aspnetcore-bootstrap/index.d.ts

```diff
@@ -0,0 +1,2495 @@
+// Type definitions for DevExpress ASP.NET 181.3
+// Project: http://devexpress.com/
+// Definitions by: DevExpress Inc. <https://github.com/devexpress>
```

Comment:
We need you to add you github id here so that you will be asked to review the changes to these definitions in future

Reply:
Ok, I'll add my id here a little later.

Reply:
@sheetalkamat , I've added my id, check it again.